### ISSUE TITLE
Admins can spawn only artificer and issues connected to that

### ISSUE THREAD

**Issue by Ratvar** [yogstation13/yogstation#2698]
Admins can spawn only artificer construct-harvester, wraith, shade, jugg are off grounds.

Artificer will look red, aka bloodcult one, while not being cult one - constructs have colour based on their affiltration, and adminspawned one get cult one automatically., and their soulstones too

**Comment by Time-Green** [comment]
Yes they can. The correct artificer is called builder something.
Juggernaut and all the others can also be spawned without any problem.

Who told you this?

**Comment by Ratvar** [comment]
Uh, apples tried, i think.

**Comment by Time-Green** [comment]
He tried to do that via the player panel?
And that guy got council?

**Comment by ShadowDeath6** [comment]
player panel is outdated and probably was forgotten about when time did his cult update
